Makefile

```diff
@@ -24,10 +24,19 @@ ENVTEST ?= setup-envtest
 GOLANGCI_LINT ?= golangci-lint
 GINKGO ?= ginkgo
 
-.PHONY: tools
-tools:
+.PHONY: all
+all: tools build
+
+.PHONY: mise
+mise:
 	@mise install --yes
 
+.git/hooks/pre-commit:
+	@pre-commit install
```

Comment:
Cool, does this target ensure `pre-commit` gets installed for users who may not have it yet?

Reply:
Yep exactly